bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BArrayType.java

```diff
@@ -128,8 +128,19 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder(elementType.toString());
-        return size != -1 ? sb.append("[").append(size).append("]").toString() : sb.append("[]").toString();
+        String array_string = "";
```

Comment:
Shall we use camelcase for all variable names

Reply:
changed in [41badd5](https://github.com/HindujaB/ballerina-lang/commit/41badd5d1af7b29c49fe440f893ccd84200a8258)